metrics/core_metrics.md

```diff
@@ -0,0 +1,82 @@
+# MDS Core Metrics
+
+The core metrics are a set of defined, consistent MDS metrics definitions that provide building blocks for other MDS [metrics](/metrics), regardless of vehicle type. 
+
+## Table of Contents
+
+- [Metrics Definitions](#metrics-definitions)
+  - [Dimensions](#dimensions)
+  - [Filters](#filters)
+  - [Special Group Type](#special-group-type)
+- [Other MDS Metrics](#other-mds-metrics)
+
+# Metrics Definitions
+
+The table below represents supported MDS core metrics and definition. All metrics are aggregated by time interval and geographic areas. This [document](https://docs.google.com/document/d/1rOhnaKWPSZApfWhFd1lzurXMbWLuZTJAYCLoxT2PQ14/edit?usp=sharing) provides methodologies and sample calculations of MDS metrics. 
```

Comment:
@whereissean I think that this document should become a markdown page in the /metrics folder.  What do you think?

Reply:
Agreed. Just needed some time to do that conversion. I think @joanathan may have started on it.

Reply:
It would be good to get it done in the next week or two since we are at the end of the release.  And upgrade it to apply to 1.0.0 (instead of 0.4.0), if anything needs to change in that regard.